build/Windows/XPlatCppWindows.vcxproj

```diff
@@ -168,7 +167,7 @@
     <ClInclude Include="$(SdkSourceDir)\include\playfab\PlayFabSpinLock.h" />
     <ClInclude Include="$(SdkSourceDir)\include\playfab\PlayFabTransportHeaders.h" />
     <ClInclude Include="$(SdkSourceDir)\include\playfab\PlayFabJsonHeaders.h" />
-    <ClInclude Include="$(SdkSourceDir)\include\playfab\QoS\DataCenterResult.h" />
+    <ClInclude Include="..\..\code\include\playfab\QoS\RegionResult.h" />
```

Comment:
making these relative paths seems weird, do we have precedence for this in the file already?

Reply:
Good catch. This is a rename from DataCenterResult to RegionResult. It doesn't change the path in a consistent way. I will fix it